scripts/zones/Uleguerand_Range/npcs/Buffalostalker_Dodzbraz.lua

```diff
@@ -1,22 +1,34 @@
 -----------------------------------
 -- Area: Uleguerand Range
---  NPC: Buffalostalker Dodzbraz
+-- NPC: Buffalostalker Dodzbraz
 -- Type: Quest NPC
+-- Quest: Bombs Away! (96)
 -- !pos -380.171 -24.89 -180.797 5
+-- !additem 1667 (Cluster Core)
+-- !additem 5267 (Shumeyo Salt)
 -----------------------------------
 local entity = {}
 
 entity.onTrade = function(player, npc, trade)
+    if npcUtil.tradeHas(trade, { { xi.items.CLUSTER_CORE, 2 } }) then -- Cluster Core x2
+        player:startEvent(8, 1667)
```

Comment:
1667 here and on line 21 should be replaced with `xi.items.CLUSTER_CORE` since that parameter is what causes the item name to display in the event